src/providers/jspm.ts

```diff
@@ -181,8 +183,28 @@ async function lookupRange (this: Resolver, registry: string, name: string, rang
     case 200:
       return { registry, name, version: (await res.text()).trim() };
     case 404:
+      const versions = await fetchVersions(name)
+      const semverRange = new SemverRange(range || '*', unstable)
+      const version = versions.find(v => semverRange.has(v))
+
+      if (version) {
+        return { registry, name, version };
```

Comment:
What error message does this lead to? It would be good to verify an example and check if it improves error usability (which is the point of this work).

Reply:
Nice, thanks for adding the error.